go/go123/Dockerfile

```diff
@@ -50,4 +50,22 @@ RUN (export GOTOOLCHAIN='auto' && \
     go install golang.org/x/lint/golint@latest && \
     go install golang.org/x/tools/cmd/goimports@latest)
 
+# Install pyenv
+RUN apt-get install -y make build-essential libssl-dev zlib1g-dev libbz2-dev \
```

Comment:
This is fine for now. But it almost feels like this should just be a different image maybe. We would not use any of these during CI, which is the main purpose for these images.

Reply:
i can make a new image